### PR TITLE
CI: apply pre-commit format fix from #49806

### PR DESCRIPTION
**Pull request by aaronsteers**
## What

This PR applies the auto-format rules from:

- https://github.com/airbytehq/airbyte/pull/49806

## Can this PR be safely reverted and rolled back?

- [ ] YES 💚
- [ ] NO ❌

### CODE REVIEW

File: .github/workflows/format-fix-command.yml [hunk 1]
```diff
@@ -67,16 +67,21 @@ jobs:
...
             [1]: ${{ steps.job-vars.outputs.run-url }}
 
-      - name: Run airbyte-ci format fix all
-        uses: ./.github/actions/run-airbyte-ci
-        continue-on-error: true
+      # Compare the below to the `format_check.yml` workflow
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: "zulu"
```
Comment (natikgadzhi) [inline .github/workflows/format-fix-command.yml:74]: Why zulu specifically?

**Comment by natikgadzhi** [comment]
@aaronsteers the format-fix run complaints at: 

> The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: [https://github.blog/changelog/2022-](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)

Should we clean this up while we're here?

File: .github/workflows/format-fix-command.yml (hunk 1 above)
Comment (aaronsteers) [reply .github/workflows/format-fix-command.yml:74]: @natikgadzhi - No reason. I copy-pasted from other workflows.

**Comment by aaronsteers** [comment]
@natikgadzhi -

> @aaronsteers the format-fix run complaints at:
> 
> > The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: [https://github.blog/changelog/2022-](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)
> 
> Should we clean this up while we're here?

- https://github.com/airbytehq/airbyte/pull/49856

**Comment by aaronsteers** [comment]
Failures appear to be related to version bumps:

`https://github.com/airbytehq/airbyte/actions/runs/12386439202/job/34574311960?pr=49852`

```
    def test_validation_pass_on_same_docker_image_tag(metadata_definition):
        success, error_message = metadata_validator.validate_docker_image_tag_is_not_decremented(metadata_definition, None)
>       assert success
E       assert False
```

Same URL and issue for all of these failures:

image
image

**Pull request merged**